charts/kube-prometheus-stack/charts/crds/templates/upgrade/clusterrole.yaml

```diff
@@ -0,0 +1,28 @@
+{{- if .Values.upgradeJob.enabled }}
```

Comment:
I think I would move upgradeJob under crds.upgradeJob to avoid potential clashes in the future. WDYT?

Reply:
Oh it is in the values but it's not used correctly here

Reply:
crd is the name of the sub chart. the values are define on top level pass to the sub chart by prefix all values with the chart name.

It works similar with grafana and node-exporter values where all grafana values prefixes with `grafana`

Reply:
Damn I'm tired, sorry about this